tests/test_api_frame.py

```diff
@@ -164,7 +161,132 @@ def test_frame_jacobians(
     assert len(frame_indices) == len(frame_names)
 
     for frame_name, frame_index in zip(frame_names, frame_indices, strict=True):
-
         J_WL_js = js.frame.jacobian(model=model, data=data, frame_index=frame_index)
         J_WL_idt = kin_dyn.jacobian_frame(frame_name=frame_name)
         assert J_WL_js == pytest.approx(J_WL_idt, abs=1e-9)
+
+
+def test_frame_jacobian_derivative(
+    jaxsim_models_types: js.model.JaxSimModel,
+    velocity_representation: VelRepr,
+    prng_key: jax.Array,
+):
+    model = jaxsim_models_types
+
+    _, subkey = jax.random.split(prng_key, num=2)
+    data = js.data.random_model_data(
+        model=model, key=subkey, velocity_representation=velocity_representation
+    )
+
+    kin_dyn = utils_idyntree.build_kindyncomputations_from_jaxsim_model(
+        model=model, data=data
+    )
+
+    # Get all names of frames in the iDynTree model.
+    frame_names = [
+        frame.name
+        for frame in model.description.frames
+        if frame.name in kin_dyn.frame_names()
+    ]
+
+    # Skip some entry of models with many frames.
+    frame_names = [
+        name
+        for name in frame_names
+        if "skin" not in name or "laser" not in name or "depth" not in name
+    ]
+
+    frame_idxs = js.frame.names_to_idxs(model=model, frame_names=tuple(frame_names))
+
+    # ===============
+    # Test against AD
+    # ===============
+
+    # Get the generalized velocity.
+    I_ν = data.generalized_velocity()
+
+    # Compute J̇.
+    O_J̇_WF_I = jax.vmap(
+        lambda frame_index: js.frame.jacobian_derivative(
+            model=model, data=data, frame_index=frame_index
+        )
+    )(frame_idxs)
+
+    assert O_J̇_WF_I.shape == (len(frame_names), 6, 6 + model.dofs())
+
+    # Compute O_J̇_WF_I using AD
```

Comment:
leftover?

Reply:
Thanks, solved in https://github.com/ami-iit/jaxsim/pull/210/commits/beed30b3a584a538fe5d97f9dad30238defda4ff